script/Arcueid/Day 7/07_00_ARC07_3_1.txt

```diff
@@ -48,7 +48,7 @@ Don't humans usually get sleepy at this time of night?"
 [sha:ccefe4a1e84c3da14c1fdd24254a5b940a5f2b20]{
 -- Page 502, Offset 17819.
 -- 「ふーん。夢は見ないの？」
-"Hmm. Do you not see any dreams?"
+"Hmm. You don't get any dreams?"
```

Comment:
'getting' dreams feels weird, maybe rephrase to use dream as verb?
> Hmm. Do you not dream or something?

Reply:
I disagree, I think this is a pretty natural expression